server/sonar-web/src/main/js/apps/issues/components/App.js

```diff
@@ -652,8 +654,36 @@ export default class App extends React.PureComponent {
     });
   };
 
-  handleIssueCheck = (issue /*: string */) => {
+  handleIssueCheck = (issue /*: string */, event /*: Event */) => {
+    // If the Shift key is pressed while (un)checking, we should (un)checked all the issues
+    // between the issue clicked and the previously clicked one
+    if (event.shiftKey && this.state.lastChecked !== null) {
+      const issueIndexes = this.state.issues.map(issue => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove curly braces and "return" from this arrow function body. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fpm%2Ffix-sonar-7449&issues=AWH_gNke9MctqhxriSg0&open=AWH_gNke9MctqhxriSg0)

[ISSUE_KEY]: AWH_gNke9MctqhxriSg0